devices/xiaomi-tissot/kernel/default.nix

```diff
@@ -0,0 +1,44 @@
+{ mobile-nixos
+, fetchFromGitHub
+, kernelPatches ? []
+, dtbTool
+, buildPackages
+}:
+let inherit (buildPackages) dtc; in
+(mobile-nixos.kernel-builder-gcc6 {
+  version = "3.18.71";
+  configfile = ./config;
```

Comment:
`config` → `config.aarch64`.

This looks dumb, since "you're not gonna run anything else than aarch64", and yes, it's likely true. Though this is to keep every kernel tree consistent. There *are* devices which will have dual-architecture builds, like `motorola-addison`.